### ISSUE TITLE
Add documentation for hosting Kratos on a sub domain

### ISSUE THREAD

**Issue by JoblersTune** [interledger/rafiki#2774]
There are specific configurations considerations and requirements when hosting Kratos on a subdomain. Need to figure out all these details and document them to make this flow easier for anyone attempting this in the future.

**Comment by JoblersTune** [comment]
Some initial considerations:
- see https://www.ory.sh/docs/kratos/reference/configuration

Within the Kratos configuration file it might be necessary to set the `domain` property on `cookies` and `sessions` for session handling and CSRF cookies.

**Comment by mkurapov** [comment]
@JoblersTune do you think we should provide an ory kratos config file in the docs for this?

**Comment by JoblersTune** [comment]
Ideally yes. But this kind of got bumped down the priority list since @tadejgolobic moved away from that approach so we haven't researched extensively or resolved the issue yet,. I.e more work than just docs still.

I think a line in the docs saying "if you host it on a sub domain read kratos docs" is sufficient for now unless it becomes a bigger priority.

**Comment by mkurapov** [comment]
> if you host it on a sub domain read kratos docs" is sufficient for now unless it becomes a bigger priority.

Makes sense, in the meantime, should we just provide a config file similar to testnet's current deployment to show a working solution? We don't need to mention sub domains.

**Comment by brad-dow** [comment]
@JoblersTune @mkurapov 
Currently, [our docs contain this brief aside and a link to the Kratos docs](https://rafiki.dev/admin/admin-user-guide/#identity-and-user-management). Should I put this issue back in the user docs backlog or are we at a point where we can build onto this?

![Image](https://github.com/user-attachments/assets/2b3ad8d6-9ab3-4632-8288-4fcbca205358)

**Comment by JoblersTune** [comment]
I think that's up to the Rafiki team. It was a priority when we were using it that way ourselves for Rafiki Money but we pivoted approaches. Personally I wouldn't make this a priority unless someone asked about it. But I also don't think it's my call any more.

@mkurapov perhaps you can provide an answer for @brad-dow.
